v2.0/enable-node-map.md

```diff
@@ -4,4 +4,197 @@ summary: Learn how to enable the node map in the Admin UI.
 toc: false
 ---
 
-*Docs coming soon*
\ No newline at end of file
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to enable the <b>Node Map</b>. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+To enable the **Node Map**, you need to start the cluster with the correct `--locality` flags and assign the latitudes and longitudes for each locality.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> won't be displayed until <i>all</i> nodes are started with the correct <code>--locality</code> flags and all localities are assigned the corresponding latitudes and longitudes. {{site.data.alerts.end}}
+
+## Example
+
+Consider a four-node geo-distributed cluster with the following configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  Node1 | us-east-1 | us-east-1a |
+|  Node2 | us-east-1 | us-east-1b |
+|  Node3 | us-west-1 | us-west-1a |
+|  Node4 | eu-west-1 | eu-west-1a |
+
+### Step 1. Ensure the CockroachDB Version is 2.0 or Higher
+
+~~~ shell
+$ cockroach version
+~~~
+
+If not, [upgrade to CockroachDB v2.0](upgrade-cockroach-version.html).
+
+### Step 2. Start (or Restart) the Nodes with the Correct `--locality` Flags
```

Comment:
Based on our conversation, what are the recommendations we want to make about the locality level to set? That's not really indicated anywhere.

Reply:
We had decided to add it as a troubleshooting item. I have added the guidance to the troubleshooting section as per our conversation.